python/cac_tripplanner/destinations/views.py

```diff
@@ -69,13 +79,9 @@ def get(self, request, *args, **kwargs):
         # make locations JSON serializable
         matched_objects = [model_to_dict(x) for x in matched_objects]
         for obj in matched_objects:
-            pnt = obj['point']
-            obj['point'] = json.loads(pnt.json)
-            img = obj.get('image')
-            if img:
-                obj['image'] = img.url
-            else:
-                obj.pop('image')
+            obj['point'] = json.loads(obj['point'].json)
+            obj['image'] = self.image_to_url(obj, 'image')
+            obj['wide_image'] = self.image_to_url(obj, 'wide_image')
```

Comment:
Will this work ok if an image field is null?  (Looks like they're both still nullable.)